### DISCUSSION TITLE
83be089e-9cad-4d55-8f77-26b1ce2d2fbc

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9572] [Rules Discussions]
# 83be089e-9cad-4d55-8f77-26b1ce2d2fbc

Learn how Figma Dev Mode streamlines developer handovers by providing a seamless bridge between design and development

https://www.ssw.com.au/rules/figma-dev-mode/

**Comment by bradystroud** [comment]
Dev mode is awesome 🚀👨‍💻
